app/src/main/res/values/strings.xml

```diff
@@ -126,6 +126,7 @@
     
     <string name="new_challenge_notification_title">New Challenge!</string>
     <string name="new_challenge_notification_message">A new challenge just got posted, check it out!</string>
+    <string name="team_chat_title">Team Chat</string>
```

Comment:
Lower case "c" for "chat" is better. See [https://m1.material.io/style/writing.html#writing-capitalization-punctuation](capitalization)